content/english/python-tensorflow/building_neural_network.md

```diff
@@ -9,27 +9,29 @@ draft: false
 
 <iframe width="100%" height="600px" src="https://www.youtube.com/embed/ttOhB-w8dt0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-This is the first step in building a neural network, which is actually a tool to allow us to do "machine learning". With this tool, we are training a model/computer to analyze various training images/data, likely labeled in advance, and perform and execute a specific task as a result. In our scenario, we are using pre-labeled images of clothing to train our model to correctly predict the type of clothing for any clothing sample we input.
+This is the first step in building a neural network, which is actually a tool that allows us to do "machine learning". With this tool, we are training a program to analyze images or data, otherwise known as training data. 
```

Comment:
```suggestion
This is the first step in building a **neural network**, which is actually a tool that allows us to do **"machine learning"**. With this tool, we are training a program to analyze images or data, otherwise known as **training data**. 
```